stylesheets/_override.shame.scss

```diff
@@ -202,3 +202,10 @@
 .lt-ie10 .content-main {
     clear: both;
 }
+
+.icon--align-with-fields {
```

Comment:
As discussed, could be replaced with font size and line height utility classes but feels like bloat for a rare situation. Leaving as is.